vignettes/wrapping-apis.Rmd

```diff
@@ -463,6 +489,7 @@ get_api_key <- function() {
   key <- Sys.getenv("NYTIMES_KEY")
   if (identical(key, "")) {
     if (is_testing()) {
+      # JENNY COMMENT HERE
```

Comment:
From the code I'm seeing here, it's not obvious to me that this condition (`is_testing()`) is sufficient to guarantee that `testing_key()` will successfully return a key. Have I missed something?

Reply:
Nope! I forgot a `secret_has_key("HTTR2_KEY")`.

Reply:
Ooops, no I didn't — `secret_decrypt()` will automatically skip if `HTTR2_KEY` is not found.